tests/hazmat/primitives/test_dh.py

```diff
@@ -201,18 +191,6 @@ def test_dh_parameters_allows_rfc3526_groups(self, backend, vector):
             # what we expect OpenSSL to have done here.
             assert serialized_params.q == (params.p - 1) // 2
 
-    @pytest.mark.skip_fips(reason="non-FIPS parameters")
-    @pytest.mark.parametrize(
-        "vector",
-        load_vectors_from_file(
-            os.path.join("asymmetric", "DH", "RFC5114.txt"), load_nist_vectors
-        ),
-    )
-    def test_dh_parameters_supported_with_q(self, backend, vector):
```

Comment:
Why are we deleting this test?

Reply:
We're deleting that because the dh_parameters_supported method was unused